AncientReligions/common/landed_titles/ACR_holy_sites.txt

```diff
@@ -1,19 +1,24 @@
-### Hellenic Pagan
-#
-#b_lebedos = {
-#	# Temple of Artemis at Ephesus
-#	holy_site = hellenic_pagan
-#}
-#b_lampsakos = {
-#	remove_holy_site = hellenic_pagan
-#}
-#b_piraeaus = {
-#	remove_holy_site = hellenic_pagan
-#}
-#b_pyrgos = {
-#	# Statue of Zeus at Olympia
-#	holy_site = hellenic_pagan
-#}
+### Hellenic Pagan Heresies (same as vanilla Hellenic)
```

Comment:
Maybe not needed if they stay the same as parent religion, because heresies inherit from parent ?